app/forms/school_onboarding_form.rb

```diff
@@ -25,7 +25,7 @@ def school_exists?
   end
 
   def school_already_onboarded?
-    if school&.try(service.to_s)
+    if school&.try("#{service}_service?".to_s)
```

Comment:
not a blocker but you probably don't need the `to_s`

Reply:
Done! Thanks for pointing that out.